src/pages/host/Host.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { Redirect, Route, Switch, NavLink as RRNavLink } from 'react-router-dom';
+import { Listing, CREATE_LISTING, GET_HOST_LISTINGS } from 'networking/listings';
+import { Container, Fade, Button, NavItem, NavLink, Nav, Row, Col } from 'reactstrap';
+import AuthenticatedRoute from 'HOCs/AuthenticatedRoute';
+import NotFound from '../notFound';
+import { compose, graphql } from 'react-apollo';
+import { getFriendlyErrorMessage } from 'utils/validators';
+
+interface Props extends RouterProps {
+  createListing: () => Promise<Listing>;
+}
+
+const Host = ({ createListing, history }: Props) => {
+  const [isCreateListingClicked, setCreateListingClicked] = React.useState<boolean>(false);
+
+  return (
+    <Container className="pt-8 pb-6" tag={Fade}>
+      <Row>
+        <Col>
+          <h1 className="mb-0">Host Profile</h1>
+        </Col>
+        <Col md="4" lg="3" xl="2" className="d-flex align-items-center justify-content-end">
+          <Button
+            block
+            disabled={isCreateListingClicked}
+            onClick={handleNewListingClick}>
+            Add New Listing
+          </Button>
+        </Col>
+      </Row>
+      <hr />
+      <Nav className="mb-5" tabs>
+        {[
+          {
+          to: '/work/host/bookings',
```

Comment:
Small whitespace irregularity here

Reply:
👀

Reply:
also, maybe can probably move this out to a `const` later?  might keep the jsx a little cleaner

Reply:
As a bonus, if we moved this to a `const`, I think we could also generate the `AuthenticatedRoute`s in the `Switch` using that same array (minor improvement. but keeps the `to` paths from falling out of sync, e.g. when we drop the `/work`)

Reply:
ahh very nice